dev-docs/bidders/addigi.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: bidder
+title: Addigi
+description: Addigi Bidder Adapter
+biddercode: addigi
+aliasCode : smarthub
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+dchain_supported: true
+media_types: banner, video, native
+safeframes_ok: true
+deals_supported: true
+floors_supported: true
+fpd_supported: false
+pbjs: true
+pbs: true
+pbs_app_supported: true
+multiformat_supported: true
```

Comment:
Please add one of the defined values here https://docs.prebid.org/formats/multiformat.html

will-bid-on-any: the adapter will consider all supported formats and bid on one or more.
will-bid-on-one: the adapter will consider only one of the supported formats for each auction.
will-not-bid: the adapter will not bid on any adunits with more than one format.

Reply:
fixed: multiformat_supported: will-bid-on-any